Add IPv6 adoption documentation (#773)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_deploying-the-bare-metal-provisioning-service.adoc

. Patch the `OpenStackControlPlane` custom resource (CR) to deploy the {bare_metal}:
+
[NOTE]

> If you are using IPv6, remember to change the load balancer IP to one that is correct in your environment. +
> E.g: `metallb.universe.tf/loadBalancerIPs:  fd00:bbbb::80`
> +
> [source,bash]

Reviewer: ```suggestion
If you use IPv6, change the load balancer IP to the IP in your environment, for example,
 `metallb.universe.tf/loadBalancerIPs:  fd00:bbbb::80`.
+
```
